evaluation/data_collator.py

```diff
@@ -0,0 +1,83 @@
+from dataclasses import dataclass
```

Comment:
nit: could you copy-paste the header (important to give back credit since we are basing this code on existing code)

Reply:
I'm unclear on that statement. Does git not track automatically those (move/copies)?